### ISSUE TITLE
Multiselect in subscribers list

### ISSUE THREAD

**Issue by christiancannata** [ibrows/IbrowsNewsletterBundle#21]
Hi,
is there a mode to select all subscribers in the list of newsletter's recipients page?
Thanks

**Comment by mikemeier** [comment]
I dont think so... But you can override the Template and make your own "Check All" Checkbox and implement the Javascript Code for that. Or you could maybe set all checked as Default (Override Controller I think)

**Comment by christiancannata** [comment]
I have tryed to modify the multiselect jquery component but there are too much subscribers in list and the js script is stops.

I think that the solution is pre-select all subscribers.

Thank you!
